epytope/TCRSpecificityPrediction/ML.py

```diff
@@ -57,15 +57,15 @@ def predict(self, tcrs, epitopes, pairwise=True, interpreter=None, conda=None, c
             epitopes = list(set(epitopes))
 
         self.input_check(tcrs, epitopes, pairwise, **kwargs)
-        data = self.format_tcr_data(tcrs, epitopes, pairwise)
+        data = self.format_tcr_data(tcrs, epitopes, pairwise,**kwargs)
         filenames, tmp_folder = self.save_tmp_files(data, **kwargs)
         cmd = self.get_base_cmd(filenames, tmp_folder, interpreter, conda, cmd_prefix, **kwargs)
         self.run_exec_cmd(cmd, filenames, interpreter, conda, cmd_prefix, **kwargs)
-        df_results = self.format_results(filenames, tcrs, epitopes, pairwise)
+        df_results = self.format_results(filenames, tcrs, epitopes, pairwise, **kwargs)
         self.clean_up(tmp_folder, filenames)
         return df_results
 
-    def format_tcr_data(self, tcrs, epitopes, pairwise):
```

Comment:
please add then kwargs to all predictors at format_results and format_tcr_data. Otherwise the model might fail, if that gets passed and but its not in the predictors functions signature